crates/proto/src/udp/udp_client_stream.rs

```diff
@@ -182,16 +184,16 @@ impl<P: RuntimeProvider> DnsRequestSender for UdpClientStream<P> {
         };
 
         let message_id = request.id();
-        let message = SerialMessage::new(bytes, self.name_server);
+        let serial_message = SerialMessage::new(bytes, self.name_server);
```

Comment:
Nit: so this should no longer be necessary, and `serial_message` feels kinda verbose/overkill for this.